department-of-veterans-affairs/abd-vro
# Validate oAuth token issued by LH

## What was the problem?

api-key was utilized to authorize version 1 of the endpoints.

Associated tickets or Slack threads:
- [MCP-2039](https://amida.atlassian.net/browse/MCP-2039)

## How does this fix it?[^1]
oAuth token will be used to authorize version 2 endpoints.

## How to test this PR
- Step 1 use the sample jwt in the secrets project to test
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
